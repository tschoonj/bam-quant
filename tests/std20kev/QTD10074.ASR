File: C:\xrfdaten\2014KW30\chris\std20kev\std10074.spe
Start: 21.07.2014 18:30:54
Stop: 21.07.2014 18:32:35
BESSY: 298.625514 298.6162853
IONI: 2.997247E-11 3.031282E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87699
Run: 100000
Dead: 12
ICR: 36520
OCR: 0
Accumulated BESSY: 59705.4693194
Accumulated Ioni: 6.03009121E-09
N Accumulation: 200
Normfactor:  3.7819125e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: W
exp_x 71.2603988647461 71.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  74   2      8.392   1224792.      1011.     210.61
