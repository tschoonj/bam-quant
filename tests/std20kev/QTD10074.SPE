$DATA:
0 2000
20636
13221
5638
1755
340
45
4
4
3
0
0
2
1
0
0
2
122
679
825
798
810
703
742
694
620
643
584
607
593
553
527
558
519
490
477
464
462
462
500
437
419
412
395
401
370
391
431
378
347
364
369
394
368
355
339
359
374
350
323
339
336
313
334
341
353
301
323
290
315
332
321
354
303
317
321
284
316
323
281
268
292
284
312
353
458
596
844
1056
1341
1323
1218
1038
782
618
484
424
305
301
303
295
314
343
308
317
343
307
296
272
276
294
271
250
266
268
264
295
272
315
274
253
259
263
264
269
276
259
236
274
267
255
280
291
265
249
247
241
259
255
234
266
235
274
264
279
286
355
341
355
327
307
277
253
239
259
262
251
237
243
269
245
253
267
249
227
263
249
230
239
264
207
230
222
218
240
240
242
254
246
225
234
228
234
206
269
214
230
245
253
225
238
251
257
240
256
231
243
245
216
222
234
271
225
256
213
233
236
246
248
257
223
247
198
259
248
239
202
239
265
238
239
234
253
235
227
235
247
224
243
245
242
229
222
209
229
244
236
230
222
243
230
252
223
255
250
249
249
241
240
252
250
246
222
260
231
280
239
257
235
255
242
229
267
240
242
277
251
248
270
238
266
257
245
264
236
250
248
294
242
278
280
282
264
237
234
247
273
273
276
271
286
313
255
258
257
275
279
260
288
284
273
269
294
244
246
304
292
287
287
256
326
316
330
302
340
327
352
311
347
318
326
334
366
335
376
401
400
467
500
482
547
516
506
480
407
397
357
346
345
318
353
306
338
332
363
356
348
378
343
363
385
402
393
381
423
435
487
593
631
827
1169
1634
2333
3117
3956
4711
5219
5390
4792
4283
3533
2582
1934
1509
1016
800
664
613
594
661
594
565
564
634
591
660
653
646
688
729
799
792
927
879
902
966
980
946
955
1040
1118
1127
1270
1478
1839
2459
3782
6073
10192
16935
27153
41585
60690
82123
103189
120705
130089
129886
120116
101845
79470
57459
38526
23964
14194
8097
4749
3147
2689
2601
2774
3056
3224
3188
3204
2892
2395
1930
1531
1201
964
832
726
732
646
672
596
649
679
623
740
687
729
730
773
817
882
979
1010
1078
1154
1274
1399
1792
2265
3132
4440
6604
8925
12245
16491
21343
27703
35431
45856
58724
73570
89496
102676
111045
113749
108887
98165
84659
70065
57343
47863
40755
36347
34357
33759
34419
35114
35158
34809
32443
28412
24180
18926
14177
10054
7057
4809
3370
2664
2217
1841
1702
1517
1415
1222
1084
879
680
531
426
294
225
208
169
163
151
154
135
161
150
156
160
162
154
167
181
155
191
168
202
252
279
355
458
546
586
650
704
689
648
659
578
601
741
957
1417
2113
3166
4464
6165
8139
10032
11205
12076
12512
11841
10274
8787
6840
5232
3831
2807
2286
1993
1963
2223
2643
3088
3401
3791
4077
4171
4288
4133
3515
3139
2421
1894
1370
982
708
538
358
303
320
323
379
422
527
625
689
807
746
731
667
537
406
316
249
155
111
72
48
30
29
14
5
16
12
14
10
7
12
11
8
3
10
15
8
9
12
8
13
7
15
7
8
8
11
13
7
7
9
14
10
7
9
9
10
7
8
12
8
7
4
6
9
11
6
10
10
8
9
9
5
6
8
7
12
8
12
5
3
8
6
4
6
7
6
14
4
6
5
7
7
5
3
8
10
8
10
6
6
8
5
5
10
7
6
8
6
8
6
3
8
9
9
6
7
5
4
4
7
8
4
5
4
4
4
4
4
7
9
7
6
4
7
5
3
2
12
6
3
9
1
8
8
6
2
9
5
7
2
9
2
6
2
7
5
9
13
6
4
7
3
7
4
6
4
5
4
6
3
3
2
5
4
3
7
7
9
8
11
4
3
8
11
19
9
27
25
27
38
38
40
30
27
35
32
27
25
17
14
13
15
10
4
3
7
8
5
8
4
1
5
6
5
8
5
8
1
7
3
6
4
10
6
3
10
4
10
18
10
12
18
26
27
32
56
53
80
98
111
102
117
127
112
116
109
94
94
84
53
46
39
26
23
25
21
14
19
15
16
27
15
14
14
12
10
10
15
11
13
16
12
12
11
11
8
15
19
16
13
8
10
17
17
9
17
10
16
20
22
31
31
31
31
40
56
65
103
85
130
145
178
179
186
208
215
222
211
195
200
194
149
150
124
111
112
85
85
95
106
64
79
67
48
55
50
36
39
25
27
22
15
19
19
13
18
14
15
14
13
10
8
13
9
10
18
10
20
16
17
15
17
27
21
25
26
30
41
42
58
66
68
73
70
94
110
116
124
143
117
127
131
144
131
128
131
113
107
128
128
121
113
133
135
145
125
147
138
102
126
137
128
119
136
142
152
192
204
190
199
208
211
181
198
207
213
171
174
155
129
117
108
86
96
84
57
45
39
31
30
22
16
13
7
10
6
6
3
4
1
5
5
2
4
2
5
3
4
7
12
15
16
11
18
16
21
19
27
20
19
19
22
29
10
18
22
20
22
13
16
18
14
21
14
9
16
18
16
16
18
10
7
8
5
10
6
1
3
3
4
4
6
3
6
2
3
5
3
2
5
1
1
2
2
2
0
2
0
0
0
0
3
0
0
0
0
1
0
0
0
0
0
0
0
1
0
1
1
0
0
0
1
1
2
2
0
0
3
0
0
2
2
0
4
1
2
0
0
0
1
2
2
1
3
0
1
0
1
2
1
0
2
0
1
0
0
1
0
1
0
0
0
0
0
0
1
2
1
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
2
0
0
2
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
1
0
1
1
0
0
1
0
1
0
1
0
1
2
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
1
0
0
1
1
0
0
0
3
1
0
0
0
0
0
2
0
1
0
0
0
1
0
2
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10074.spe
Start: 21.07.2014 18:30:54
Stop: 21.07.2014 18:32:35
BESSY: 298.625514 298.6162853
IONI: 2.997247E-11 3.031282E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87699
Run: 100000
Dead: 12
ICR: 36520
OCR: 0
Accumulated BESSY: 59705.4693194
Accumulated Ioni: 6.03009121E-09
N Accumulation: 200
Normfactor:  3.7819125e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: W
 
exp_x 71.2603988647461 71.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  74   2      8.392   1224792.      1011.     210.61
