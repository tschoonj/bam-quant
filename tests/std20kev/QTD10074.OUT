     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:54  บ   
     บ  Spectrum: QTD10074.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        5 iterations done    บ   
     บ  ChiSquare =    98.8   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -20.0 ๑   100.0   บ     -20.0 ๑      .5   บ   
     บ  GAIN (eV/ch)  บ     20.202 ๑   2.000   บ    20.202 ๑    .001   บ   
     บ det NOISE (eV) บ       70.2 ๑    40.0   บ      70.2 ๑     3.1   บ   
     บ  FANO factor   บ       .138 ๑    .050   บ      .138 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 W -L1                            336021. ๑    863.           บ   
     บ    L1M3        9.819    .41627      139874. ๑    360.   1170.9  บ   
     บ              487.035    184.25         187.           9.76E-01  บ   
     บ    L1M2        9.525    .32896      110537. ๑    284.    572.1  บ   
     บ              472.482    181.87         195.           9.74E-01  บ   
     บ    L1N3       11.674    .11300       37971. ๑     98.    697.0  บ   
     บ              578.858    198.57         158.           9.85E-01  บ   
     บ    L1N2       11.608    .08415       28276. ๑     74.    808.5  บ   
     บ              575.591    198.08         160.           9.85E-01  บ   
     บ    L1O3       12.059    .03006       10101. ๑     29.     56.3  บ   
     บ              597.916    201.42         148.           9.83E-01  บ   
     บ    L1M5       10.291    .01442        4846. ๑     18.     25.8  บ   
     บ              510.399    188.00         174.           9.79E-01  บ   
     บ    L1M4       10.228    .00961        3230. ๑     16.    122.0  บ   
     บ              507.281    187.50         176.           9.78E-01  บ   
     บ    L1N5       11.851    .00353        1186. ๑     13.    332.9  บ   
     บ              587.620    199.88         153.           9.86E-01  บ   
     บ                                                                 บ   
     บ  2 W -L2                           1152051. ๑   1225.           บ   
     บ    L2M4        9.672    .80449      926818. ๑    986.    704.6  บ   
     บ              479.758    183.07         191.           9.75E-01  บ   
     บ    L2N4       11.286    .16044      184833. ๑    197.   1819.5  บ   
     บ              559.652    195.66         168.           9.84E-01  บ   
     บ    L2M1        8.724    .02168       24971. ๑     30.    146.4  บ   
     บ              432.832    175.25         219.           9.66E-01  บ   
     บ    L2N1       10.949    .00553        6376. ๑     15.     29.3  บ   
     บ              542.970    193.10         177.           9.82E-01  บ   
     บ    L2O4       11.539    .00615        7089. ๑     15.   1046.5  บ   
     บ              572.176    197.56         161.           9.85E-01  บ   
     บ    L2O1       11.468    .00101        1162. ๑     13.   1451.7  บ   
     บ              568.661    197.03         163.           9.84E-01  บ   
     บ    L2M3        9.261    .00070         803. ๑     14.    829.6  บ   
     บ              459.413    179.72         203.           9.71E-01  บ   
     บ                                                                 บ   
     บ  3 W -L3                           1540797. ๑   1272.           บ   
     บ    L3M5        8.398    .71392     1100011. ๑    908.    210.6  บ   
     บ              416.694    172.47         229.           9.62E-01  บ   
     บ    L3M4        8.335    .08099      124782. ๑    104.    165.1  บ   
     บ              413.576    171.93         231.           9.61E-01  บ   
     บ    L3N5        9.960    .15201      234215. ๑    194.   1197.8  บ   
     บ              494.014    185.38         183.           9.77E-01  บ   
     บ    L3M1        7.388    .03668       56514. ๑     49.    168.1  บ   
     บ              366.699    163.59         229.           9.45E-01  บ   
     บ    L3N1        9.612    .00920       14173. ๑     18.    709.0  บ   
     บ              476.788    182.58         193.           9.74E-01  บ   
     บ    L3O1       10.129    .00162        2501. ๑     14.    621.4  บ   
     บ              502.380    186.72         178.           9.78E-01  บ   
     บ    L3O5       10.200    .00558        8601. ๑     15.    279.3  บ   
     บ              505.895    187.28         177.           9.78E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                           11016. ๑    107.           บ   
     บ    L3N1       18.064    .24756        2759. ๑     28.      3.0  บ   
     บ              895.146    241.49          51.           9.78E-01  บ   
     บ    L3O1       16.783    .16360        1823. ๑     19.      4.6  บ   
     บ              831.765    233.52          65.           9.86E-01  บ   
     บ    L3O5       19.344    .09365        1044. ๑     12.      7.6  บ   
     บ              958.528    249.20          35.           9.62E-01  บ   
     บ               18.352    .06257         697. ๑     10.      1.8  บ   
     บ              909.402    243.25          47.           9.75E-01  บ   
     บ    L2N1       19.678    .04937         550. ๑      8.     32.4  บ   
     บ              975.040    251.17          31.           9.57E-01  บ   
     บ    L3M5       19.632    .04734         528. ๑      8.     18.2  บ   
     บ              972.784    250.90          32.           9.58E-01  บ   
     บ    L3N5       18.211    .03739         417. ๑      8.      2.7  บ   
     บ              902.423    242.39          49.           9.77E-01  บ   
     บ    L3M1       20.958    .03735         416. ๑      6.      7.1  บ   
     บ             1038.422    258.60          20.           9.30E-01  บ   
     บ    L3N1       17.926    .03333         371. ๑      8.      5.0  บ   
     บ              888.359    240.65          53.           9.79E-01  บ   
     บ               19.491    .02829         315. ๑      7.      8.7  บ   
     บ              965.804    250.07          34.           9.60E-01  บ   
     บ               19.207    .02522         281. ๑      7.      8.4  บ   
     บ              951.740    248.39          37.           9.65E-01  บ   
     บ               20.024    .01991         222. ๑      6.    133.2  บ   
     บ              992.212    253.21          28.           9.51E-01  บ   
     บ               15.780    .01510         168. ๑      9.      6.9  บ   
     บ              782.087    227.08          81.           9.88E-01  บ   
     บ               21.305    .01506         168. ๑      4.      7.2  บ   
     บ             1055.593    260.57          17.           9.22E-01  บ   
     บ               17.060    .01142         127. ๑      8.      2.9  บ   
     บ              845.468    235.27          60.           9.85E-01  บ   
     บ               21.246    .00944         105. ๑      4.      7.5  บ   
     บ             1052.678    260.24          17.           9.23E-01  บ   
     บ               19.779    .00715          80. ๑      6.     68.5  บ   
     บ              980.060    251.77          30.           9.55E-01  บ   
     บ               17.116    .00667          74. ๑      8.      2.3  บ   
     บ              848.220    235.62          59.           9.84E-01  บ   
     บ               19.495    .00637          71. ๑      6.      8.7  บ   
     บ              965.996    250.10          34.           9.60E-01  บ   
     บ               19.920    .00598          67. ๑      5.    116.3  บ   
     บ              987.040    252.60          29.           9.53E-01  บ   
     บ               21.105    .00564          63. ๑      4.      8.8  บ   
     บ             1045.698    259.44          18.           9.27E-01  บ   
     บ               18.610    .00512          57. ๑      7.      1.7  บ   
     บ              922.196    244.81          44.           9.73E-01  บ   
     บ               18.396    .00505          56. ๑      7.      1.8  บ   
     บ              911.601    243.52          47.           9.75E-01  บ   
     บ               20.821    .00503          56. ๑      5.      3.8  บ   
     บ             1031.634    257.81          21.           9.34E-01  บ   
     บ               19.890    .00388          43. ๑      5.    104.1  บ   
     บ              985.578    252.42          29.           9.53E-01  บ   
     บ               19.354    .00381          42. ๑      6.      7.6  บ   
     บ              959.017    249.26          35.           9.62E-01  บ   
     บ               21.593    .00381          42. ๑      4.      5.0  บ   
     บ             1069.849    262.20          16.           9.14E-01  บ   
     บ               22.572    .00372          42. ๑      3.      3.1  บ   
     บ             1118.315    267.66           9.           8.86E-01  บ   
     บ               22.919    .00300          33. ๑      3.      2.3  บ   
     บ             1135.487    269.57           7.           8.75E-01  บ   
     บ               17.348    .00289          32. ๑      8.      3.5  บ   
     บ              859.724    237.07          56.           9.83E-01  บ   
     บ               20.451    .00270          30. ๑      5.     19.6  บ   
     บ             1013.304    255.68          24.           9.42E-01  บ   
     บ               18.674    .00228          25. ๑      7.      2.6  บ   
     บ              925.362    245.20          43.           9.72E-01  บ   
     บ               21.452    .00228          25. ๑      4.      6.7  บ   
     บ             1062.870    261.40          16.           9.18E-01  บ   
     บ               19.638    .00214          24. ๑      6.     21.6  บ   
     บ              973.081    250.94          32.           9.58E-01  บ   
     บ               21.731    .00204          23. ๑      4.      3.9  บ   
     บ             1076.685    262.98          15.           9.10E-01  บ   
     บ               21.168    .00203          23. ๑      4.      8.5  บ   
     บ             1048.806    259.79          18.           9.25E-01  บ   
     บ               17.207    .00173          19. ๑      8.      1.9  บ   
     บ              852.745    236.19          58.           9.84E-01  บ   
     บ               19.341    .00170          19. ๑      6.      7.6  บ   
     บ              958.358    249.18          35.           9.63E-01  บ   
     บ               19.070    .00170          19. ๑      6.      9.6  บ   
     บ              944.953    247.57          38.           9.67E-01  บ   
     บ               16.923    .00154          17. ๑      8.      5.3  บ   
     บ              838.681    234.40          62.           9.85E-01  บ   
     บ               20.621    .00129          14. ๑      5.      2.3  บ   
     บ             1021.740    256.66          22.           9.38E-01  บ   
     บ               18.684    .00128          14. ๑      7.      2.6  บ   
     บ              925.857    245.26          43.           9.72E-01  บ   
     บ               20.010    .00101          11. ๑      5.    130.8  บ   
     บ              991.495    253.12          28.           9.51E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
